cmd/analyzer/main.go

```diff
@@ -34,7 +34,7 @@ func init() {
 func main() {
 	flag.Parse()
 	repoName = flag.Arg(0)
-	if flag.NArg() > 1 || repoName == "" || launchDir == "" {
+	if flag.NArg() > 1 || repoName == "" {
```

Comment:
Empty launch dir isn't valid here.

Reply:
Yes that was the thought. We were very inconsistent about which values we were validating weren't empty. I decided to go with none. An invalid dir should be caught at a later I think.

Reply:
👍